sections/03_categorize.md

```diff
@@ -1,23 +1,43 @@
-## Does deep learning create a strategic inflection point in how we categorize individuals with respect to health and disease?
```

Comment:
Your pull request included some older commits.  I'll focus on the single most recent one.